TUT/STA130F24_TUT06_Oct25.ipynb

```diff
@@ -0,0 +1,12137 @@
+       "                            \n",
+       "var gd = document.getElementById('730f7145-01f1-4b0d-9f1a-b8cc8a0b7e9e');\n",
+       "var x = new MutationObserver(function (mutations, observer) {{\n",
+       "        var display = window.getComputedStyle(gd).display;\n",
+       "        if (!display || display === 'none') {{\n",
+       "            console.log([gd, 'removed!']);\n",
+       "            Plotly.purge(gd);\n",
+       "            observer.disconnect();\n",
+       "        }}\n",
+       "}});\n",
+       "\n",
+       "// Listen for the removal of the full notebook cells\n",
+       "var notebookContainer = gd.closest('#notebook-container');\n",
+       "if (notebookContainer) {{\n",
+       "    x.observe(notebookContainer, {childList: true});\n",
+       "}}\n",
+       "\n",
+       "// Listen for the clearing of the current output cell\n",
+       "var outputEl = gd.closest('.output');\n",
+       "if (outputEl) {{\n",
+       "    x.observe(outputEl, {childList: true});\n",
+       "}}\n",
+       "\n",
+       "                        })                };                });            </script>        </div>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "broadway = pd.read_csv('broadway.csv')\n",
+    "drop = broadway.isna().any()\n",
+    "columns = drop[drop].index\n",
+    "broadway.drop(columns = columns, axis = 1, inplace = True)\n",
+    "import plotly.express as px\n",
+    "fig = px.scatter(x= broadway.weekly_gross_overall, y = broadway.avg_ticket_price, trendline= 'ols')\n",
+    "fig.show()"
+   ]
```

Comment:
So I'll take this code and decide if/how I might want to use it... maybe it's indeed a part of this TUT or maybe a part of my LEC, or maybe it ends up fully out of scope for this version of STA130. 

- So this the cell where you should put your SLR hypothesis testing demo